target/linux/ath79/dts/qca9557_kt_dw02-412h.dtsi

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "KT DW02-412H (64M)";
+	compatible = "kt,dw02-412h-64m", "qca,qca9557";
+
+	aliases {
+		led-boot = &led_wan;
+		led-failsafe = &led_wan;
+		led-running = &led_wan;
```

Comment:
If you want to use this LED for WAN, you should drop this line.

Reply:
There are no other LEDs left for these functions.
So I'm concerned that at least the failsafe status cannot be shown by LED.
Is there any alternative method to keep this function work?

Reply:
I think we can use the LED for boot/failsafe/upgrade purpose (as they are special situations) but we cannot use it for WAN and running at the same time (though in practice running will be overridden by WAN). By dropping *this* line I meant only `led-running`.

Reply:
Ah, I misundertood it as meaning to delete all these lines, sorry for my bad English...